nvdaHelper/liblouis/sconscript

```diff
@@ -59,8 +59,10 @@ env['CC'] = 'clang-cl'
 if 'analyze' in env['nvdaHelperDebugFlags']:
 	env.Append(CCFLAGS='/analyze-')
 
-# Ignore some warnings as the code is not ours
-env.Replace(CCFLAGS='/W2')

```

Comment:
should this be removed? why?

Reply:
Ah I'm sorry, I motivated this slightly https://github.com/nvaccess/nvda/pull/14313#issuecomment-1297028520 but forgot to mention it here.
The short story is that we are not treating warnings as errors for liblouis, so there is no point in changing the warning level, especially since with and without the warning level replacement, the number of warnings is equal anyway.